nodes/config/ui_base.html

```diff
@@ -322,13 +358,19 @@
             refreshSidebarEditorDebounced()
         }
     })
-    RED.events.on('nodes:add', function (event) {
-        if (RED._db2debug) { console.log('nodes:add', event) }
-        if (event.dirty && event.type && event.type.startsWith('ui-')) {
+    RED.events.on('nodes:add', function (node) {
+        if (RED._db2debug) { console.log('nodes:add', node) }
+        if (node.dirty && node.type && node.type.startsWith('ui-')) {
             if (RED._db2debug) { console.log('nodes:add - this is a ui- node! queuing a call to refreshSidebarEditors') }
             // debounce the call to refreshSidebarEditors as multiple events can be fired in quick succession
             refreshSidebarEditorDebounced()
         }
+
+        // if we're adding a ui-base
+        if (node.type.startsWith('ui-')) {
+            // action on all ui- elements to ensure we've remapped (now) missing ui-base nodes
+            checkDuplicateUiBases()
+        }
```

Comment:
@joepavitt ^

Reply:
@joepavitt ok, I tested the debounce with 1ms and it does work fine. That is because the import process is synchronous.  However, that is a node-red think (not something dashboard has a say in) and therefore may not always be the case.  I will update the suggestion to 25ms.  that is too quick for anyone to paste an import and operate the deploy.

Reply:
Ready for you to review + apply now @joepavitt